### DISCUSSION TITLE
What tool is used for building monospace fonts? Or if there's any tutorial or course that could help me out?

### DISCUSSION THREAD

**Discussion by ritasrfaria** [tonsky/FiraCode#1256] [Q&A]
Hi! 👋 I'm a graphic designer and I love your font! I'm interested in learning more about monospace fonts and I was wondering what tool you used to build Fira Code? Or if there's any tutorial or course that could help me out?

**Comment by tonsky** [comment]
Hi! Monospaced fonts are built just like any other fonts, as far as I know.

In case of Fira Code we use Glyphs.app. I don’t know good tutorials, but they have [a good learning section](https://glyphsapp.com/learn) on their website. There must be something on YouTube, I’m sure, too.
